pkg/interfaces/contracts/standalone-utils/ICowSwapFeeBurner.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IERC1271 } from "@openzeppelin/contracts/interfaces/IERC1271.sol";
+
+import { ICowConditionalOrder, GPv2Order } from "../solidity-utils/misc/ICowConditionalOrder.sol";
+import { ICowConditionalOrderGenerator } from "../solidity-utils/misc/ICowConditionalOrderGenerator.sol";
+import { IProtocolFeeBurner } from "./IProtocolFeeBurner.sol";
+
+interface ICowSwapFeeBurner is IERC1271, IProtocolFeeBurner, ICowConditionalOrder, ICowConditionalOrderGenerator {
+    enum OrderStatus {
+        NotExist,
+        Active,
+        Filled,
+        Failed
+    }
+
+    error InvalidOrderParameters(string reason);
+    error NonZeroOffchainInput();
+    error OrderHasUnexpectedStatus(OrderStatus actualStatus);
+    error InterfaceIsSignatureVerifierMuxer();
+
+    event OrderRetried(IERC20 sellToken, uint256 sellAmount, uint256 minTargetTokenAmount, uint256 deadline);
+    event OrderReverted(IERC20 sellToken, address receiver, uint256 sellAmount);
```

Comment:
Events go above errors, and everything in a public interface should have NatSpec.